<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsWindow</class>
 <widget class="QMainWindow" name="StatsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StatsWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="configTab">
     <attribute name="title">
      <string>Страница</string>
     </attribute>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>450</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Start server</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="portEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>33033</string>
      </property>
     </widget>
     <widget class="QLabel" name="portLbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="annoyingLbl">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>kekeke, a lot of settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="messagesTab">
     <attribute name="title">
      <string>Сообщения</string>
     </attribute>
     <widget class="QTextEdit" name="logBrowser">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>0</y>
        <width>476</width>
        <height>488</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="dbTab">
     <attribute name="title">
      <string>БД</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>460</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>clients</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="watchTableButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>460</y>
        <width>95</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
